src/app/core/record-works/record-works.service.ts

```diff
@@ -206,7 +206,7 @@ export class RecordWorksService {
       )
   }
 
-  save(work: Work, bibtex?: boolean) {
+  save(work: Work, bibtex?: boolean): Observable<any> {
```

Comment:
Please define a response object. 
Maybe a simple generic error object is ok like
```suggestion
  save(work: Work, bibtex?: boolean): Observable<{ errors :string}> {
```